Gaijinizer patch/Map081.txt

```diff
@@ -207,63 +223,68 @@ creature" that we burned in our organization too
 せいぜい足掻いてくれたまえ……
 > CONTEXT: Map081/events/1/pages/0/129/Dialogue/J_People1
 > CONTEXT: Map081/events/1/pages/1/186/Dialogue/J_People1
-It is not sweet like two people so far ...
-At best you scratch me ...#MTLed
+It won't be quite as soft on you as these last
+two...
+Do your best...#Ron
```

Comment:
Minor flow:
>_**It won't be quite as soft on you as the last
two you've faced...**_
>
>_**Do your best... \\\N[1]-kun.**_

(Adding the name here might be a bit of a localization, but I think it adds to it.)